client/src/test/omtLinkProvider.test.ts

```diff
@@ -0,0 +1,62 @@
+import { DocumentLink, commands, workspace, Uri, Position, Range } from 'vscode';
+import { equal, deepEqual } from 'assert';
+import { getDocUri, activate } from './helper';
+
+describe('Should make links of imports', () => {
+	const docUri = getDocUri('one/imports.omt');
+	const expectedCount = 6;
+	let actualDocumentLinks: DocumentLink[];
+
+	before(async () => {
+		await activate(docUri);
+		actualDocumentLinks = (await commands.executeCommand(
+			'vscode.executeLinkProvider',
+			docUri,
+		)) as DocumentLink[];
+	});
+
+	it('total link count', () => {
+		equal(actualDocumentLinks.length, expectedCount);
+	})
+
+	describe('uri imports', () => {
+		it('should make a link or a relative path', () => {
```

Comment:
"a link to", "... of", or "... from" maybe? Not sure what you mean, but probably not "or"

Reply:
'for a relative path'